packages/eas-cli/src/commands/build/view.ts

```diff
@@ -0,0 +1,87 @@
+import { getConfig } from '@expo/config';
+import { Command } from '@oclif/command';
+import { HTTPError } from 'got';
+import ora from 'ora';
+
+import { apiClient } from '../../api';
+import { Build } from '../../build/types';
+import formatBuild from '../../build/utils/formatBuild';
+import log from '../../log';
+import {
+  findProjectRootAsync,
+  getProjectAccountNameAsync,
+  getProjectIdAsync,
+} from '../../project/projectUtils';
+
+export default class BuildView extends Command {
+  static description = 'view a build for your project';
+
+  static args = [{ name: 'buildId' }];
+
+  async run() {
+    const { buildId } = this.parse(BuildView).args;
+
+    const projectDir = (await findProjectRootAsync()) ?? process.cwd();
+    const projectId = await getProjectIdAsync(projectDir);
+    const accountName = await getProjectAccountNameAsync(projectDir);
+    const { exp } = getConfig(projectDir, { skipSDKVersionRequirement: true });
+    const projectName = `@${accountName}/${exp.slug}`;
```

Comment:
There's a similar if not the same block of code for `build:list`. Maybe it'd be worth creating a util for this?